uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/README.md

```diff
@@ -17,114 +31,65 @@ Installation notes
 Step 1 - Dependencies
 ---------------------
 
-This provider uses the *KeyCloak Admin Client* (which uses KeyCloak's REST API services) to manage the different entities, so the following keycloak artifacts are required:                         
+This provider uses a custom RestEasy client that consumes the Keycloak remote admin services, so the following JAXRS, RestEasy 
```

Comment:
The readme needs an update here. It's not clear which provider is meant here ("this provider"). I can infer that it's likely the credentials provider, but it should be made clear with a heading or subsection.